src/CodeGen.fs

```diff
@@ -1195,12 +1231,142 @@ type GraphqlErrorResponse = {{ errors: {errorType} list }}
+
+        /// <summary>
+        /// Creates {clientName} specifying <see href="T:HttpClient">HttpClient</see> instance
+        /// </summary>
+        /// <param name="url">GraphQL endpoint URL</param>
+        /// <remarks>
+        /// In order to enable all F# types serialization and deserealization
+        /// <see href="T:System.Text.Json.Serialization.JsonFSharpConverter">JsonFSharpConverter</see> by default is added
+        /// from <a href="https://github.com/Tarmil/FSharp.SystemTextJson">FSharp.SystemTextJson</a> NuGet package
+        /// </remarks>
+        new: url: string * client: HttpClient -> {clientName}
+
+        /// <summary>Creates {clientName}</summary>
+        /// <param name="url">GraphQL endpoint URL</param>
+        /// <remarks>
+        /// In order to enable all F# types serialization and deserealization
+        /// <see href="T:System.Text.Json.Serialization.JsonFSharpConverter">JsonFSharpConverter</see> by default is added
+        /// from <a href="https://github.com/Tarmil/FSharp.SystemTextJson">FSharp.SystemTextJson</a> NuGet package
+        /// </remarks>
+        new: url: string -> {clientName}
+
+        /// <summary>Creates {clientName} specifying <see href="T:HttpClient">HttpClient</see> instance</summary>
+        /// <remarks>
+        /// In order to enable all F# types serialization and deserealization <b>you must</b> add
+        /// <see href="T:System.Text.Json.Serialization.JsonFSharpConverter">JsonFSharpConverter</see>
+        /// from <a href="https://github.com/Tarmil/FSharp.SystemTextJson">FSharp.SystemTextJson</a> NuGet package yourself
+        /// </remarks>
+        /// <param name="url">GraphQL endpoint URL</param>
+        new: client: HttpClient * options: JsonSerializerOptions -> {clientName}
+
+        /// <summary>Creates {clientName}</summary>
+        /// <remarks>
+        /// In order to enable all F# types serialization and deserealization
+        /// <see href="T:System.Text.Json.Serialization.JsonFSharpConverter">JsonFSharpConverter</see> by default is added
+        /// from <a href="https://github.com/Tarmil/FSharp.SystemTextJson">FSharp.SystemTextJson</a> NuGet package
+        /// </remarks>
+        new: client: HttpClient -> {clientName}
+    end
+"""
+
+let private sampleFSharpNewtonsoftGraphqlClientFsi projectName clientName =
+        $"""namespace {projectName}
+
+open Newtonsoft.Json
+open Newtonsoft.Json.Linq
+open Fable.Remoting.Json
+open System.Net.Http
+open System.Text
+
+type {clientName} =
+    class
+        /// <summary>Creates {clientName} specifying <see href="T:HttpClient">HttpClient</see> instance</summary>
+        /// <remarks>
+        /// In order to enable all F# types serialization and deserealization
+        /// <see href="T:Fable.Remoting.Json.FableJsonConverter">FableJsonConverter</see> is added
+        /// from <a href="https://github.com/Zaid-Ajaj/Fable.SimpleJson">Fable.SimpleJson</a> NuGet package
```

Comment:
Link pointing to different repo, it has to point to https://github.com/Zaid-Ajaj/Fable.Remoting because that is where `Fable.Remoting.Json` lives